### PR TITLE
Add Wurm Unlimited

### PR DESCRIPTION
**Pull request by IceOfWraith**
This is the very start of the Wurm Unlimited template. It allows starting the server and joining with manual setting input. I have been slow to progress it, so collaboration is welcome.

### CODE REVIEW

File: wurm-unlimitedconfig.json [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+        "DisplayName": "Custom Java Arguments",
+        "Category": "Server Settings",
+        "Description": "Settings that will be appended to the Java arguments for the server. Only change this if you know what you're doing!",
+        "Keywords": "custom,java,arguments,args",
+        "FieldName": "JvmParam0",
+        "InputType": "text",
+        "IsFlagArgument": false,
+        "ParamFieldName": "VMParams.JvmParam0",
+        "IncludeInCommandLine": false,
+        "DefaultValue": "-XX:+AggressiveOpts",
```
Comment (Greelan) [inline wurm-unlimitedconfig.json:197]: Remove comma at end of line

File: wurm-unlimitedconfig.json [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+        "DisplayName": "Custom Java Location",
+        "Category": "Server Settings",
+        "Description": "If you receive \"Failed to load jvm.dll\", you may specify a custom location where the jvm.dll is located. Only change this if you know what you're doing!",
+        "Keywords": "custom,java,arguments,args,location",
+        "FieldName": "JavaPath",
+        "InputType": "text",
+        "IsFlagArgument": false,
+        "ParamFieldName": "Runtime.JavaPath",
+        "IncludeInCommandLine": false,
+        "DefaultValue": "C:\\Program Files\\Java\\jre1.8.0_321\\bin\\server",
```
Comment (Greelan) [inline wurm-unlimitedconfig.json:225]: Remove comma at end of line

File: wurm-unlimitedmetaconfig.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "ConfigFile": "./LaunchConfig.ini",
+    "AutoMap": true,
+    "ConfigType": "ini",
```
Comment (Greelan) [inline wurm-unlimitedmetaconfig.json:5]: Remove comma at end of line

File: wurm-unlimitedconfig.json (hunk 1 above)
Comment (IceOfWraith) [reply wurm-unlimitedconfig.json]: ,

File: wurm-unlimitedconfig.json (hunk 2 above)
Comment (IceOfWraith) [reply wurm-unlimitedconfig.json]: ,

File: wurm-unlimitedmetaconfig.json (hunk 3 above)
Comment (IceOfWraith) [reply wurm-unlimitedmetaconfig.json]: ,

**Pull request merged**